modules/op-tkn-trigger-management.adoc

```diff
@@ -40,6 +40,15 @@ List EventListeners.
 $ tkn eventlistener list -n myspace
 ----
 
+== eventlistener logs
+List EventListener logs.
```

Comment:
`Display logs of a EventListener`
https://docs.openshift.com/container-platform/4.7/cli_reference/tkn_cli/op-tkn-reference.html#pipelinerun-logs